spec/student_spec.rb

```diff
@@ -29,7 +32,8 @@
   end
 
   describe "#create_table" do
-    it 'creates the students table in the database' do
+    it 'creates the students table in the database', :skip_before do
+      DB[:conn].execute("DROP TABLE students")
```

Comment:
Interesting -- I'm not totally up on SQLite behavior, but my one worry is that it'd throw an error if the table *didn't* exist when this test runs (if, for example, the student runs this test on its own).